exp_mimic/prompts/age.txt

```diff
@@ -1,15 +1,15 @@
 Here is a clinical note:
 "{note}"
-Output a list of descriptors that summarizes the patient case (such as aspects on demographics, diagnoses, SDOH, etc). For each descriptor, also list as many descriptors that mean the same thing or generalizations of the descriptor. All descriptors, synonyms, and generalizations cannot be more than two words. Output as a JSON in the following format. Do not output more than 30 entries in the JSON:
+Output a list of descriptors that summarizes the patient case (such as aspects on demographics, diagnoses, SDOH, etc). Output as a JSON in the following format where each value is a one word descriptor. Do not output more than 30 entries in the JSON:
```

Comment:
can you make a new prompt file? this will make sure we keep the code submitted to AISTATS separate

Reply:
maybe we can even create a new experiment folder for this synonym experiment